File: SGCOS.Domain/Equipamento.cs
using System;
using System.Collections.Generic;

namespace SGCOS.Domain
{
    public class Equipamento
    {
        public int Id { get; set; }
        public string NrSerie { get; set; }
        public string Descricao { get; set; }
        public string Marca { get; set; }
        public string Modelo { get; set; }
        public string img { get; set; }
        public int ClienteId { get; set; }
        public Cliente Cliente { get; }
        public List<Servico> Servicos { get; set; }

    }
}

File: SGCOS-App/src/app/_models/Equipamento.ts
import { Servico } from './Servico';

export interface Equipamento {
    id: number;
    nrSerie: string;
    descricao: string;
    marca: string;
    modelo: string;
    imagemURL: string;
    clienteId: number;
    servicos: Servico[];
}
